src/org/mozilla/javascript/Parser.java

```diff
@@ -3272,7 +3272,10 @@ private AstNode arrayLiteral()
                     elements.add(new EmptyExpression(ts.tokenBeg, 1));
                     skipCount++;
                 }
-            } else if (tt == Token.RB) {
+            }else if(tt == Token.COMMENT) {
+                consumeToken();
+            }
+            else if (tt == Token.RB) {
```

Comment:
Same here -- It looks like other code in this file puts "else" on the same line as the preceding brace, with a space between the brace and else...

Reply:
Noted. Will fix the formatting and push the changes.